Makefile

```
@@ -5,7 +5,7 @@ init:
 	pip install -Ur requirements.txt
 
 build:
-	build-image-src/build_all_images.sh
+	cd build-image-src; ./build_all_images.sh
```

nit: similar style to other commands?
```suggestion
	cd build-image-src
	./build_all_images.sh
```

---

this doesn't work in makefile - https://stackoverflow.com/questions/1789594/how-do-i-write-the-cd-command-in-a-makefile

---

Interesting behavior, wasn't aware!